### ISSUE TITLE
Add Database Button

### ISSUE THREAD

**Issue by davidperezgar** [Automattic/studio#568]
### What

As a developer, we need a button for accessing easily to the database. I know that is SQLite, but a direct access and open with a default client DB in the dashboard.

### Why

We use a lot the Database manager.

### How

![Captura de pantalla 2024-09-27 a las 9 10 09](https://github.com/user-attachments/assets/c4e1010d-c6c4-42ba-b4c5-777c48a2ac83)

**Comment by wojtekn** [comment]
Thanks for adding the feature request @davidperezgar .

> We use a lot the Database manager.

What app are you using to manage the database?

**Comment by davidperezgar** [comment]
Thanks. Good question. Normally we use Sequel, TablePlus or Adminer.

**Comment by talldan** [comment]
https://github.com/Automattic/studio/issues/63 is related. That allows selecting editor/terminal. It won't be scalable to add every kind of application, so at some point there might need to be a way to add custom paths or an 'Open in ...' button.

**Comment by mikaoelitiana** [comment]
This seems to be a duplicate of #420

**Comment by wojtekn** [comment]
@mikaoelitiana yes, and soon we will solve it by https://github.com/Automattic/studio/pull/2834 🥳

**Comment by wojtekn** [comment]
We've added phpMyAdmin support in Studio 1.7.7.

I'm closing this issue.
